installer-guide/mac-install.md

```diff
@@ -5,73 +5,40 @@
 
 ## Downloading macOS: Modern OS
 
 * This method allows you to download macOS 10.13 and newer, for 10.12 and older see [Downloading macOS: Legacy OS](#downloading-macos-legacy-os)
 
 From a macOS machine that meets the requirements of the OS version you want to install, go directly to the App Store and download the desired OS release and continue to [**Setting up the installer**](#setting-up-the-installer).
 
-For machines that need a specific OS release or can't download from the App Store, you can use the gibMacOS utility.
+For machines that need a specific OS release or can't download from the App Store, you can use the Munki's InstallInstallMacOS utility.
 
-Now let's grab [gibMacOS](https://github.com/corpnewt/gibMacOS), then unzip it into a local directory.
+In order to run it, just copy and paste the below command in a terminal window:
 
-Next run the `gibMacOS.command`:
+`curl -O https://raw.githubusercontent.com/munki/macadmin-scripts/main/installinstallmacos.py && sudo /usr/bin/python installinstallmacos.py`
 
-![](../images/installer-guide/mac-install-md/gib.png)
 
-As you can see, we get a nice list of macOS installers. If you need beta versions of macOS, you can select `C. Change Catalog`. For this example we'll choose 1:
+![](../images/installer-guide/mac-install-md/munki.png)
 
-![](../images/installer-guide/mac-install-md/gib-process.png)
+As you can see, we get a nice list of macOS installers. If you need a particular versions of macOS, you can select it by typing the number next to it. For this example we'll choose 10:
+
+![](../images/installer-guide/mac-install-md/munki-process.png)
 
 * **macOS 11, Big Sur Note**: As this OS is quite new, there's still some issues with certain systems to resolve. For more information, see here: [OpenCore and macOS 11: Big Sur](../extras/big-sur/README.md)
   * For first time users, we recommend macOS 10.15, Catalina
 * **Nvidia GPU Note**: Reminder to verify whether your hardware support newer OSes, see [Hardware Limitations](../macos-limits.md)
 
 This is going to take a while as we're downloading the entire 8GB+ macOS installer, so it's highly recommended to read the rest of the guide while you wait.
 
-Once finished, we have to either extract the installer or build it:
-
-* [Extracting the installer](#extracting-the-installer)
-  * For macOS 11+
-* [Building the installer](#building-the-installer)
-  * For 10.15 and older
-
-### Extracting the installer
-
-For macOS 11 and newer, Apple now packages up the installer into the InstallAssistant package. This will be located under `gibMacOS/macOS Downloads/`:
-
-![](../images/extras/big-sur/readme/final-download.png)
-
-Run the InstallAssistant.pkg and point this to whichever drive you're booting off of, this is where the Install.app will be dropped:
+Once finished, you'll find in your `Users` folder a DMG containing the macOS Installer, called like `Install_macOS_11.1-20C69.dmg`. Mount it and you'll find the installer application.
```

Comment:
As mentioned earlier, maybe don't use the base `~/` folder to keep things cleaner

Reply:
The developer recommends to use `/Users/Shared` or similar

Reply:
User's base folder(`~/`) isn't protected, so macOS shouldn't care either